macros/contexts/contextSignificantFigures.pl

```diff
@@ -316,25 +323,6 @@ sub perl {
 	return $self->context->Package('Real') . '->new(' . $value->value . ',' . $value->N . ')';
 }
 
-=head2 SigFigNumber
-
-Create a number directly (instead using Compute) either with or without the number of significant
-digits.
-
-Usage:
-
-    SigFigNumber('1.2345');
-	SigFigNumber(10000,3); # makes a sig fig number which is 1.00 * 10**4
-
-=cut
-
-sub SigFigNumber {
-	return context::SignificantFigures::Number->new(@_x);
-}
-
-sub eval {
-	$self = shift;
-	return $self->SUPER::eval(@_);
-}
+sub eval { shift->SUPER::eval(@_) }
```

Comment:
I don't understand this function, as it does nothing but call the original `eval()`, so removing it would only make things more efficient.